### ISSUE TITLE
Scanning for nearby devices doesn't seem to work.

### ISSUE THREAD

**Issue by Jambajoo** [thaliproject/postcardapp#120]
I've successfully built story_001_dadougla branch. I got it installed on my phones. 

Phone 1 : Samsung Note 5 
Phone 2 : Samsung Galaxy S4

However, I tried scanning for nearby devices and it seems like it doesn't work at all. It just keeps on showing me the Finding icon rotating animation. It's still loading in the moment I type this issue btw. haha

**Comment by deadlyfingers** [comment]
@tongkolizer Yes, the Identity Exchange part has a known discovery/finding issue - but if you want to try it out I have documented the process I used to get it to work on the issue: https://github.com/thaliproject/postcardapp/issues/97 - but just bear in mind the Identity Exchange part is still in development. The Identity Exchange process is used to authenticate a peer so you know a postcard has been shared by that user, but if both your Android devices are running Thali you will automatically be able to share postcards between your Android devices.
